### ISSUE TITLE
Every weekday (monday to friday) option?

### ISSUE THREAD

**Issue by Plupuu** [insanum/gcalcli#472]
How do I create an event every weekday (monday to friday), for example, every day at 6pm? I wanted to create an event for every day, but apparently there is no such option. Sorry if it already exists, I did not find :(

**Comment by jcrowgey** [comment]
Sounds like a nice feature.  I don't think we have any support for recurring at this point.

**Comment by ZaxonXP** [comment]
Indeed, it would be nice to handle creating recurring events. Waiting for this feature as well. :)
